omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -202,14 +253,72 @@ code from the project's GitHub account to build locally::
     :doc:`/developers/build-system`
         Developer documentation page on how to build the OMERO.server
 
+3. Once the build completes, the OMERO server build output will be located in :file:`~/Projects/Omero/code/projects/openmicroscopy/dist`.
+   Prepend the :file:`bin` directory to your :envvar:`PATH`::
+
+    $ export PATH=~/code/projects/openmicroscopy/dist/bin:$PATH
 
+   and follow the steps for setting up the database and OMERO data directory as mentioned in the previous section.
```

Comment:
What previous section ? there is no such section in this doc, which would deal with OMERO data directory creation and setting up the database.

Reply:
In fact, I cannot find any reference in the doc which would lead me now to the steps of creating a new local db and /OMERO directory at all. this is definitely missing here

Reply:
As discussed, the section is actually below (not "previous") and is called OMERO configuration.